src/reducers/elf.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
It was added with help from Jonas because the elf-file signature was unknown, and eslint was giving errors everywhere. This got rid of most of them, but there's still one remaining when assigning to the sections-variable in canvas.tsx